setup.py

```diff
@@ -43,6 +43,12 @@ def get_long_description():
     author="A lot of people",
     author_email="code-quality@python.org",
     url="https://github.com/PyCQA/pyflakes",
+    project_urls={
+        "Source": "https://github.com/PyCQA/pyflakes",
```

Comment:
this duplicates the `url` that's already there -- I don't think this adds any value